scanpipe/pipes/d2d.py

```diff
@@ -664,14 +673,19 @@ def map_about_files(project, logger=None):
     for about_file_resource in from_about_files:
         _map_about_file_resource(project, about_file_resource, to_resources)
 
+        about_file_companions = about_file_resource.siblings().filter(
+            name__contains=about_file_resource.name_without_extension
```

Comment:
`__startswith` may be more precise. To be confirmed.

Reply:
Yes, this would be more precise, adding this. Thanks!